app/main.py

```diff
@@ -24,9 +39,27 @@ def profile(request: Request):
     # Get relevant data from database
     upcouming_events = range(5)
     current_username = "Chuck Norris"
-
     return templates.TemplateResponse("profile.html", {
         "request": request,
         "username": current_username,
         "events": upcouming_events
     })
+
+
+# sessions = SessionLocal()  # instance of db
+
+
+@app.post("/emailbackground")
```

Comment:
Probably better to move to a specific route file.

Reply:
+1, shouldn't be on main.py

Reply:
Just send_email, we can see it uses async